meta-resin-common/recipes-go/resin-provisioner/resin-provisioner_1.0.4.bb

```diff
@@ -9,6 +9,9 @@ GO_IMPORT = "github.com/resin-os/resin-provisioner"
 
 inherit binary-compress
 FILES_COMPRESS = "${bindir}/resin-provision"
+# FIXME upx fails to compress resin-provision on Aarch64
+# upx: /usr/bin/resin-provision: UnknownExecutableFormatException
+FILES_COMPRESS_aarch64 = ""
```

Comment:
This was actually a known issue (or support lack) and we deactivated in the past in the machine specific layer (ex https://github.com/resin-os/resin-artik710/blob/master/layers/meta-resin-artik710/recipes-go/resin-provisioner/resin-provisioner_1.0.4.bbappend). But I like this being fixed at this level.